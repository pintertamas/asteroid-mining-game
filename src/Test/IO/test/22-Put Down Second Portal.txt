putPortalDown
move;0
putPortalDown