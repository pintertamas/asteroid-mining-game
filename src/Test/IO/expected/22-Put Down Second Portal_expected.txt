successful;putPortalDown
successful;move;0
successful;putPortalDown